lib/app.jsx

```diff
@@ -84,6 +87,28 @@ const isElectron = ( () => {
 	return () => foundElectron;
 } )();
 
+const when = ( condition ) => ( action, fallback = noop ) => ( ... args ) => (
+	condition( ... args ) ? action( ... args ) : fallback( ... args )
+)
```

Comment:
I couldn't find anywhere here that we use `fallback`, so I wonder if we couldn't leave it out and make this simpler.

```js
const when = condition => action => ( ...args ) => condition( ...args ) ? action : null;
```

I'm going to add some comments about this below, too, because I'm not sure if we need it.